webapp/app/js/directives/clonejobs.js

```diff
@@ -1069,6 +1175,84 @@ treeherder.directive('thCloneJobs', [
             tableEl.append(row);
         }
     };
+    
+    var traceParent = function(el, level){
+
+        var i;
+        for (i=0; i<level; i++) {
+            el = el.parent();
+        }
+
+        return el;
+    }
+
+    var traceJob = function(elem, step, clss){
+
+        var elemsList, jobFound, jobElem, reverseList, tempList, jobsList;
+
+        elemsList = [elem];
+        jobFound = false;
+        jobElem = null;
+        reverseList = false;
+
+        while (!jobFound && elemsList.length>0) {
```

Comment:
A single character space is needed on either side of the `>` comparison operator.